src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -24,6 +24,20 @@
 )
 
 
+def save(trainer, save_path):
+    """Save reward model and policy."""
+    os.makedirs(save_path, exist_ok=True)
+    th.save(trainer.reward_trainer.model, os.path.join(save_path, "reward_net.pt"))
+    if hasattr(trainer.trajectory_generator, "algorithm"):
+        serialize.save_stable_model(
+            os.path.join(save_path, "policy"),
+            trainer.trajectory_generator.algorithm,
+            trainer.vec_normalize,
```

Comment:
Why not have `save` take `vec_normalize` directly? It's created inside the script, and not used by the trainer class at all.

Reply:
Good point. I missed out that previously.